packages/wifi_iot/ios/Classes/SwiftWifiIotPlugin.swift

```diff
@@ -121,7 +121,7 @@ public class SwiftWifiIotPlugin: NSObject, FlutterPlugin {
     private func connect(call: FlutterMethodCall, result: @escaping FlutterResult) {
         let sSSID = (call.arguments as? [String : AnyObject])?["ssid"] as! String
         let _ = (call.arguments as? [String : AnyObject])?["bssid"] as? String? // not used
-        let sPassword = (call.arguments as? [String : AnyObject])?["password"] as! String?
+        let sPassword = (call.arguments as? [String : AnyObject])?["password"] as? String? ?? nil
```

Comment:
also is `?? nil` necessary?

Reply:
If you watch the dart side of this function you are providing default values to other parameters.
`{
    String? password,
    NetworkSecurity security = NetworkSecurity.NONE,
    bool joinOnce = true,
    bool withInternet = false,
    bool isHidden = false,
  }`

Reply:
If you watch the dart side of this function you are providing default values to other parameters.
{ String? password, NetworkSecurity security = NetworkSecurity.NONE, bool joinOnce = true, bool withInternet = false, bool isHidden = false, } so it won't crash on other params because they always exist.


"also is ?? nil necessary?"
Programically it is useless, but because later usage of sPassword variable it wouldn't compile without direct cast to nil. It would have required bigger refactoring to not use it that way.

Reply:
Ok. Thanks for explaining.